api/v1/models/proxy_statistics.go

```diff
@@ -17,7 +17,7 @@ import (
 
 // ProxyStatistics Statistics of a set of proxy redirects for an endpoint
 // swagger:model ProxyStatistics
-// +k8s:deepcopy-gen=true
+
```

Comment:
Is this deliberately removed? (Check for equivalent changes across all files)

Reply:
No, I'll fix it up. I've pushed the branch primarily to make sure the code is not lost.